frontend/util.lua

```diff
@@ -99,6 +99,91 @@ function util.secondsToClock(seconds, withoutSeconds)
     end
 end
 
+
+-- Converts seconds to a period of time string.
+
+---- @int seconds number of seconds
+---- @bool withoutSeconds if true 1h30', if false 1h30'10''
+---- @bool hmsFormat, if true format 1h30m10s
+---- @treturn string clock string in the form of 1h30' or 1h30'10''
+function util.secondsToHClock(seconds, withoutSeconds, hmsFormat)
```

Comment:
Seems to never be called with `hmsFormat=true` ? Did you mean for this to be a setting?

Reply:
Not used yet. For future use :)